docs/events/spokane-rust-kickoff.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Spokane Rust Kick-Off Announcement"
+description: "Be a part of the first Spokane Rust User Group meetup!" 
+---
+
+<!-- index start -->
+
+# Spokane Rust Kick-Off
+
+_March 22nd, 2024_ | [Spokane Rust on Meetup! :fontawesome-brands-meetup:](https://www.meetup.com/spokane-rust/)
+
+![](../img/spokane_rust_launch.png)
+
+## Rustaceans and Programming Enthusiasts in the Eastern Washington area!
+
+We are thrilled to announce our inaugural meetup kicking off Spokane Rust, our local Rustlang user group! Join us for an evening of Rust camaraderie, learning, and discussion on March 22nd, 2024 at 6:00PM, taking place at the IntelliTect office 1720 W. 4th Ave. Unit B, Spokane, WA 99201.
```

Comment:
Are you planning your meetups on Friday evenings? That might be a difficult day of the week for a lot of people to make.

Reply:
I'm not certain yet - I was thinking it might be a better idea to throw out a "prospective date" then refine it as we near that. I think second or third Thursdays might work a bit better.